src/main/c/Makefile

```diff
@@ -1,17 +1,119 @@
-.PHONY: all test clean libsimdpp
+# This makefile invokes itself recursively to build the libraries,
+# possibly with different compiler options.
+#
+# If compiling with g++, we can specify the abi-version to keep
+# maximum compatibility when linked against old versions of libstdc++
+#   -fabi-version=2 was the default from g++-3.4 to g++-4.9
+#   -fabi-version=9 was the default in g++-5.x
```

Comment:
I think the comments are out of date now.

Reply:
I think they're in step with the code as it is, but no longer apply after we take out all the abi-version and recursive-invocation stuff.  So the comment will be removed.